physionet-django/events/views.py

```diff
@@ -146,6 +147,43 @@ def event_home(request):
         else:
             form_error = True
 
+    event_details = {}
+    for selected_event in events_all:
+        participants = selected_event.participants.all()
+        pending_applications = selected_event.applications.filter(
+            status__in=[EventApplication.EventApplicationStatus.WAITLISTED])
+        rejected_applications = selected_event.applications.filter(
+            status__in=[EventApplication.EventApplicationStatus.NOT_APPROVED])
+        withdrawn_applications = selected_event.applications.filter(
+            status__in=[EventApplication.EventApplicationStatus.WITHDRAWN])
+
+        event_details[selected_event.id] = [
+            {
+                'id': 'participants',
+                'title': 'Total participants:',
+                'count': participants.count(),
```

Comment:
I'm just thinking that maybe this would be faster? `len` doesn't require an additional query and the results are going to be small here.
```suggestion
                'count': len(participants),
```